pyramid/pyramid.go

```diff
@@ -0,0 +1,22 @@
+package pyramid
+
+// StorageType is enumeration of the different tiered storage types
+// supported by pyramid.
+type StorageType int
+
+const (
+	StorageTypeSSTable StorageType = iota
+	StorageTypeTreeManifest
```

Comment:
I think comments here are needed.  E.g.: is this the tier?  Are high numbers slower?

Or is it the file type?  In that case, I'd want the _client_ to control the type just by holding one pyramid for each time.  (Consider the case of a RefsManager managing commits)